### ISSUE TITLE
Crash when erasing empty file

### ISSUE THREAD

**Issue by ElXreno** [curlpipe/ox#63]
**Describe the bug**
If you try to erase an empty file, `ox` will crash.

**To Reproduce**
1. Run `ox`
2. Press `backspace` few times
3. See error

**Expected behavior**
No crash.

**Screenshots**
Not required.

**Desktop (please complete the following information):**
 - OS: Fedora 33 (KDE)

**Crash log**
```
thread 'main' panicked at 'index out of bounds: the len is 0 but the index is 0', src/document.rs:385:21                                                           
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace                                                                                                                                                     
fish: “ox” terminated by signal SIGABRT (Abort)
```

**Comment by curlpipe** [comment]
Thanks for the report, I'll work on fixing this in the next version.

**Comment by curlpipe** [comment]
I've patched this in my 0.2.7 branch, which is for development. Hopefully I can merge and release 0.2.7 into master in 1 or 2 weeks time.

**Comment by ElXreno** [comment]
Thanks for the quick fix!
